src/main/java/org/protege/editor/owl/client/diff/ui/ReviewButtonsPanel.java

```diff
@@ -244,7 +251,20 @@ public void actionPerformed(ActionEvent e) {
         	 
         }
     };
-    
+
+    private ActionListener restartPelletBtnListener = new ActionListener() {
+        @Override
+        public void actionPerformed(ActionEvent actionEvent) {
+            ClientSession sess = ClientSession.getInstance(editorKit);
+            LocalHttpClient client = (LocalHttpClient) sess.getActiveClient();
+
+            if (!client.isWorkFlowManager(sess.getActiveProject())) return;
+
+            // TODO: implement this
+//            pelletService.restart();
```

Comment:
@rgrinberg Injecting it naively runs into the problem bob was mentioning (depending on pellet-client)
@bdionne agreed. It isn't really related to any"Changes" from the changes view. Maybe calling it ```restart remote reasoner``` to specify that it doesn't apply to the local or incremental reasoners.